### PR TITLE
netch: Fix default configuration

### PR DESCRIPTION
**Pull request by chawyehsu**
Missing initial field settings causes app exception.

### CODE REVIEW

File: bucket/netch.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "extract_dir": "Netch",
     "pre_install": [
         "ensure \"$dir\\data\" | Out-Null",
-        "if (!(Test-Path \"$persist_dir\\data\\settings.json\")) { Set-Content \"$dir\\data\\settings.json\" '{ \"CheckUpdateWhenOpened\": false }' -Encoding Ascii -Force }"
+        "if (!(Test-Path \"$persist_dir\\data\\settings.json\")) { Set-Content \"$dir\\data\\settings.json\" '{\"CheckUpdateWhenOpened\":false,\"Server\":[]}' -Encoding Ascii -Force }"
```
Comment (Ash258) [inline bucket/netch.json:15]:
```suggestion
        "if (!(Test-Path \"$persist_dir\\data\\settings.json\")) { Set-Content \"$dir\\data\\settings.json\" '{ \"CheckUpdateWhenOpened\": false, \"Server\": [] }' -Encoding Ascii -Force }"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/netch.json (hunk 1 above)
Comment (chawyehsu) [reply bucket/netch.json:15]: spaces make no sense, contents of `settings.json` will be overwritten when the app start.

Comment (Ash258) [reply bucket/netch.json:15]: There is no reason to provide ugly minified json

Comment (chawyehsu) [reply bucket/netch.json:15]: There is no reason to add meaningless spaces to a one-line JSON string which will be overwritten

Comment (Ash258) [reply bucket/netch.json:15]: It is not about the final file, but about the maintainability of the manifest.

**Pull request merged**